templates/visuals/table/src/settings.ts

```diff
@@ -25,7 +25,27 @@
  */
 "use strict";
 
-import { dataViewObjectsParser } from "powerbi-visuals-utils-dataviewutils";
-import DataViewObjectsParser = dataViewObjectsParser.DataViewObjectsParser;
+import { formattingSettings } from "powerbi-visuals-utils-formattingmodel";
 
-<%= visualSettingsClasses %>
+import FormattingSettingsModel = formattingSettings.Model;
+// import FormattingSettingsCard = formattingSettings.Card;
+// import FormattingSettingsSlice = formattingSettings.Slice;
+
+// TODO: fill all visual settings here
+// class DataPointCardSettings extends FormattingSettingsCard {
```

Comment:
Comments are taken from produced "table" custom visual after running `pbiviz new -t table` and converted to match new formatting model